js/core/options/index.js

```diff
@@ -166,7 +167,7 @@ export class Options {
     }
 
     initial(name) {
-        return this._initial[name];
+        return getNestedOptionValue(this._initial, name);
```

Comment:
Why not to use `compileGetter`?

Reply:
It turns the function into the result of its call

Reply:
@ksercs,

I think he is talking about something like:

```jsx
const cachedGetters = {}; // maybe should rename this

initial(name) {
  cachedGetters[name] = cachedGetters[name] || compileGetter(name);
  return cachedGetters[name](this._initial, { functionsAsIs: true });
}
```
(like in optionManager)

Reply:
oh, we can configure compileGetter by `functionAsIs`.. Thanks!

Reply:
fixed it